server/generator.js

```diff
@@ -0,0 +1,62 @@
+const faker = require("faker")
+
+const generateUser = (id, teamId = undefined) => {
```

Comment:
Setting the default value to an argument to undefined is a bit redundant and is pretty much the same of not setting the default at all

```suggestion
const generateUser = (id, teamId) => {
```